ue5osc/__init__.py

```diff
@@ -41,9 +41,17 @@ def send_and_await(self, osc_address: str):
         return self.message_handler.wait_for_response()
 
     def get_project_name(self) -> str:
-        """Returns and optionally prints the name of the current connected project."""
+        """Returns the name of the current connected project."""
         return self.send_and_await("/get/project")
 
+    def get_raycast(self) -> str:
+        """Returns the length of the raycast."""

```

Comment:
This documentation should probably also describe the use case since it is not obvious compared to the other methods.